addons/l10n_in_edi/models/account_move.py

```diff
@@ -225,7 +229,32 @@ def _l10n_in_edi_send_invoice(self):
                         "doc_date": self.invoice_date and self.invoice_date.strftime("%d/%m/%Y"),
                     }
                 )
-                if not response.get("error"):
+                mismatch_error = []
+                if jwt:
+                    try:
+                        if SignedInvoice := response.get("data", {}).get('SignedInvoice', {}):
+                            decoded_response = jwt.decode(SignedInvoice, options={'verify_signature': False})
+                            decoded_response = json.loads(decoded_response.get('data', '{}'))
+                            received_gstin = decoded_response.get("BuyerDtls", {}).get('Gstin')
+                            expected_gstin = generate_json.get("BuyerDtls", {}).get('GSTIN')
+                            received_total_invoice_value = decoded_response.get("ValDtls", {}).get('TotInvVal')
+                            expected_total_invoice_value = generate_json.get("ValDtls", {}).get('TotInvVal')
+                            # Check for mismatch between decoded and expected e-invoice details:
+                            # - Buyer GSTIN matches
+                            # - Total Invoice Value is within ±1 (to allow rounding differences)
```

Comment:
Rounding difference should trigger a mismatch  😬
In any case this check is way too permissive.

Reply:
this is base on rule of Government API 
![image](https://github.com/user-attachments/assets/ce802284-4cc1-4f01-9d52-aa2c463cb6be)

Reply:
Thanks for the info ! 
Fine by me then. Could we please get a comment to explain this ?